docusaurus/docs/get-started/anatomy-of-a-plugin.md

```diff
@@ -0,0 +1,181 @@
+---
+id: anatomy-of-a-plugin
+title: Anatomy of a plugin
+description: This guide describes the anatomy of a plugin, including the individual components that make up each plugin type.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - create-plugin
+  - folders
+  - anatomy
+  - components
+  - apps
+  - data sources
+  - panels
+sidebar_position: 10
+---
+
+# Anatomy of a plugin
+
+Grafana plugins enable you to extend Grafana’s core functionality by adding custom features, such as new data sources, visualizations, or entire applications. Each plugin type caters to specific needs, allowing developers to tailor solutions for a variety of use cases, from integrating external data services to creating bespoke UI elements.
+
+This guide will walk you through the essential components of a Grafana plugin, the folder structure generated by the `create-plugin` tool, and how to organize your code for different plugin types. By the end, you will have a clear understanding of how to develop and structure a plugin, utilizing both frontend and backend components.
+
+Before proceeding, we recommend reviewing the [plugin types and usage guide](/key-concepts/plugin-types-usage) to gain a basic understanding of the different types of plugins available.
+
+## Plugin types
+
+Each Grafana plugin is composed of several essential components that extend Grafana’s functionality in different ways. In this section, we’ll explore the core parts of a plugin, focusing on the three primary plugin types: Apps, Data Sources, and Panels.
```

Comment:
```suggestion

Each Grafana plugin is composed of several essential components that extend Grafana’s functionality in different ways. First, we’ll explore the core parts of each of the three primary plugin types: Apps, Data Sources, and Panels.
```

This is H2 but so are App/DS/Panel Plugins which seems odd... However, if we drop App/DS/Panel to H3 then we'll lose the header links in the right menu for the individual features. I've tried to reword to remove the heading entirely.

Reply:
Yeah, ideally I wanted App/DS/Panel to live under this, but ran into the same nav bar issue, so ended up just keeping them at the same H2 level. Happy to just remove the heading.